examples/patterns/navigation/sidebar.html

```diff
@@ -0,0 +1,138 @@
+          <li><a class="sidebar__link" href="#">Second level 3 <i class="p-icon--plus"></i><i class="p-icon--minus"></i></a>
+            <ul class="sidebar__third-level is-deepest-level">
+              <li><a class="sidebar__link" href="#">Third level 1</a></li>
+              <li><a class="sidebar__link" href="#">Third level 2</a></li>
+              <li><a class="sidebar__link" href="#">Third level 3</a></li>
+              <li><a class="sidebar__link" href="#">Third level 4</a></li>
+              <li><a class="sidebar__link" href="#">Third level 5</a></li>
+            </ul>
+          </li>
+          <li><a class="sidebar__link" href="#">Second level 4</a></li>
+          <li><a class="sidebar__link" href="#">Second level 5</a></li>
+        </ul>
+      </li>
+      <li><a class="sidebar__link" href="#">Section title 3 <i class="p-icon--plus"></i><i class="p-icon--minus"></i></a>
+        <ul class="sidebar__second-level">
+          <li><a class="sidebar__link" href="#">Second level 1</a></li>
+          <li><a class="sidebar__link" href="#">Second level 2</a></li>
+          <li><a class="sidebar__link is-selected" href="#">Second level 3 <i class="p-icon--plus"></i><i class="p-icon--minus"></i></a>
+            <ul class="sidebar__third-level is-deepest-level">
+              <li><a class="sidebar__link" href="#">Third level 1</a></li>
+              <li><a class="sidebar__link" href="#">Third level 2</a></li>
+              <li><a class="sidebar__link" href="#">Third level 3</a></li>
+              <li><a class="sidebar__link" href="#">Third level 4</a></li>
+              <li><a class="sidebar__link" href="#">Third level 5</a></li>
+            </ul>
+          </li>
+          <li><a class="sidebar__link" href="#">Second level 4</a></li>
+          <li><a class="sidebar__link" href="#">Second level 5</a></li>
+        </ul>
+      </li>
+    </ul>
+    <hr>
+    <ul class="p-list">
+      <li><a class="p-link--external sidebar__link" href="#">Supplementary link 1</a></li>
+      <li><a class="p-link--external sidebar__link" href="#">Supplementary link 2</a></li>
+    </ul>
+  </div>
+</div>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illum ducimus maiores, sunt sequi officia neque sit? Error blanditiis modi mollitia, reprehenderit id illum! Quas suscipit officia vitae voluptatum dolores, facere?</p>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illum ducimus maiores, sunt sequi officia neque sit? Error blanditiis modi mollitia, reprehenderit id illum! Quas suscipit officia vitae voluptatum dolores, facere?</p>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illum ducimus maiores, sunt sequi officia neque sit? Error blanditiis modi mollitia, reprehenderit id illum! Quas suscipit officia vitae voluptatum dolores, facere?</p>
+
+<script>
+  // Sidebar menu overlay toggle
+  let toggleElements = document.querySelectorAll('[data-js="sidebar-toggle"]');
```

Comment:
We should hook javascript to classes as detailed in the practices: https://github.com/canonical-webteam/practices/blob/master/coding/stylesheets.md#javascript-namespacing

If you believe targeting data attributes is better then please create a proposal on the practises document first.